internal/app/client/client.go

```diff
@@ -119,26 +140,25 @@ func invoke(ctx context.Context, req *v1alpha1.ClientCompatRequest) (*v1alpha1.C
 		clientOptions = append(clientOptions, connect.WithProtoJSON())
 	}
 
-	// TODO - How do we configure each compression algo? i.e.
-	// how do we know the string to use and the func for WithCompression?
+	// TODO - Add support for other compression algos
 	switch req.Compression {
 	case v1alpha1.Compression_COMPRESSION_GZIP:
 		clientOptions = append(clientOptions, connect.WithSendGzip())
-	case v1alpha1.Compression_COMPRESSION_IDENTITY:
 	case v1alpha1.Compression_COMPRESSION_BR:
 	case v1alpha1.Compression_COMPRESSION_ZSTD:
 	case v1alpha1.Compression_COMPRESSION_DEFLATE:
 	case v1alpha1.Compression_COMPRESSION_SNAPPY:
+	case v1alpha1.Compression_COMPRESSION_IDENTITY:
+		// Do nothing
 	case v1alpha1.Compression_COMPRESSION_UNSPECIFIED:
 		// Do nothing
 	}
 
-	var wrapper Wrapper
 	switch req.Service {
 	case conformancev1alpha1connect.ConformanceServiceName:
-		wrapper = NewConformanceClientWrapper(transport, serverURL, clientOptions)
+		wrapper := NewInvoker(transport, serverURL, clientOptions)
+		return wrapper.Invoke(ctx, req)
```

Comment:
small nit -- var is still named "wrapper" which no longer really makes sense. Also, var isn't needed
```suggestion
		return NewInvoker(transport, serverURL, clientOptions).Invoke(ctx, req)
```